core/metadata/metadata.go

```diff
@@ -5,6 +5,13 @@ import (
 	"github.com/vmihailenco/msgpack/v5"
 )
 
+const (
```

Comment:
I recommend moving these 4 constants back to the `core/metadata` module. Afterward, declare the functions `SetMetadataTID` and `SetMetadataSID` to set the `tid` and `sid`.